lang/resources/org/partiql/type-domains/partiql.ion

```diff
@@ -668,6 +688,13 @@ may then be further optimized by selecting better implementations of each operat
                 // rows for which `predicate` returns true.
                 (filter predicate::expr source::bexpr)
 
+                (aggregate
```

Comment:
Worth to leave some comments. I think the point that we treat `GROUP AS` as a special aggregate function is quite a big change and may worth to explain here.